### PR TITLE
Report errors if the file is not found.

### PR DESCRIPTION
**Pull request by sam-falvo**

### CODE REVIEW

File: file_info.py [hunk 1]
```diff
@@ -36,15 +36,19 @@ def main():
...
+        age = int(time.time() - details.st_ctime)
+        size = details.st_size
+        mode = oct(details.st_mode & 0777)
+
+        print "status ok target exists"
+        print "metric age int", age
+        print "metric bytes int", size
+        print "metric mode string", mode
+    except Exception e:
+        print "status error {}".format(str(e))
```
Comment (jjbuchan) [inline file_info.py]: Got an example for what str(e) will print?

Maybe you'd want to distinguish between a few of the more "expected" exceptions?

Comment (jjbuchan) [inline file_info.py]: ...and by distinguish, I mean specify your own wording for the reason

Comment (sam-falvo) [inline file_info.py]: It basically is the result of perror() on the ERRNO value.  Here's an example of what comes back for me: `"[Errno 2] No such file or directory: 'aksdjhfsdf'"`

**Comment by jjbuchan** [comment]
+1 (if anyone else has strong feelings about catching specific exceptions they can make subsequent PRs)

**Pull request merged**